tests/ui/extern/extern-types-size_of_val.rs

```diff
@@ -10,6 +12,7 @@ extern "C" {
 fn main() {
     let x: &A = unsafe { &*(1usize as *const A) };
 
+    // These don't have a dynamic size, so this should panic.
     assert_eq!(size_of_val(x), 0);
     assert_eq!(align_of_val(x), 1);
```

Comment:
This doesn't get to the second call. Maybe use `// run-pass` + `catch_unwind().unwrap_err()`? (or multiple revisions)

Reply:
`catch_unwind` doesn't work, this is a non-unwinding panic.